diff --git a/scripts/install_openssl.sh b/scripts/install_openssl.sh
index 91f90df..5e3145b 100755
--- a/scripts/install_openssl.sh
+++ b/scripts/install_openssl.sh
@@ -40,7 +40,6 @@ ${ARCH_PROG} ./config --prefix=${INSTALLDIR} \
                       --libdir=lib \
                       --debug \
                       enable-fuzz-libfuzzer \
-                      --with-fuzzer-lib=/usr/lib/libFuzzingEngine \
                       -DPEDANTIC \
                       -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION \
                       no-shared \
